tests/drum/test_custom_model.py

```diff
@@ -63,6 +75,78 @@
 WEIGHTS_CSV = "weights-csv"
 
 
+class DrumServerProcess:
+    def __init__(self):
+        self.process = None
+        self.out_stream = None
+        self.err_stream = None
+
+    @property
+    def returncode(self):
+        return self.process.returncode
+
+
+class DrumServerRun:
+    def __init__(
+        self, framework, problem, language, docker=None, custom_model_dir=None, force_start=False,
+    ):
+        self._custom_model_dir_to_remove = None
+        if custom_model_dir is None:
+            # If custom model directory is not set explicitly - it's is created for the class.
+            # In such case it should also be removed.
+            custom_model_dir = TestCMRunner._create_custom_model_dir(framework, problem, language)
+            self._custom_model_dir_to_remove = custom_model_dir
+
+        port = 6799
```

Comment:
Maybe we should use `CMRunnerUtils.find_free_port()` here?

Reply:
I'll leave this as is for this PR - it works and has been working for quite some time.

Reply:
we can improve this separately